api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -126,17 +126,20 @@ public void validate(Object obj, Errors errors) {
 			}
 			
 			
-			if (!StringUtils.isBlank(user.getEmail())) {
-				if(!isEmailValid(user.getEmail())) {
+			if (!StringUtils.isBlank(user.getEmail()) && !isEmailValid(user.getEmail())) {
+
```

Comment:
Try not to leave such blank lines, especially if there is only one command in the braces.

Reply:
Finished that

Reply:
Anything else @sacull

Reply:
@dkayiwa  feel free to merge this ka PR  though i bumped into it and we close it